sdformat_to_mjcf/sdformat_to_mjcf/converters/model.py

```diff
@@ -34,21 +34,29 @@ def add_model(mjcf_root, model):
     model_pose = graph_resolver.resolve_pose(model.semantic_pose())
 
     def convert_node(body, node):
-        child_body = add_link(body,
-                              node.link,
-                              node.parent_node.link.name())
+        if not is_sdf_world:
+            child_body = add_link(body,
+                                  node.link,
+                                  node.parent_node.link.name())
+        else:
+            pose = model.raw_pose()
+            child_body = add_link(body,
+                                  node.link,
+                                  node.parent_node.link.name(),
+                                  pose=pose)
 
         add_joint(child_body, node.joint)
 
         for cn in node.child_nodes:
             convert_node(child_body, cn)
 
     for cn in kin_hierarchy.world_node.child_nodes:
-        # Adjust the poses of each of the nodes to account for the model
-        link_pose = graph_resolver.resolve_pose(cn.link.semantic_pose())
-        new_link_pose = model_pose * link_pose
-        cn.link.set_raw_pose(new_link_pose)
-        cn.link.set_pose_relative_to("")
+        if not is_sdf_world:
+            # Adjust the poses of each of the nodes to account for the model
+            link_pose = graph_resolver.resolve_pose(cn.link.semantic_pose())
+            new_link_pose = model_pose * link_pose
+            cn.link.set_raw_pose(new_link_pose)
```

Comment:
Not sure if we need the whole logic of `is_sdf_world` and adding a pose to `add_link` since we are resolving the pose of the link to the parent of model here. Since we currently don't support nested models, `new_link_pose` here is the pose of the link relative to world, so when `add_link` is called, `node.link` already has the world pose of the link. 

Maybe a better variable name and a comment is needed here.

Reply:
https://github.com/gazebosim/gz-mujoco/pull/37/commits/855f32f670558f87132b0d33902d6009af583127